eva-server/src/main/java/uk/ac/ebi/eva/server/ws/VariantWSServerV2.java

```diff
@@ -60,10 +61,18 @@
     private VariantWithSamplesAndAnnotationsService service;
 
     @GetMapping(value = "/{variantCoreString}")
-    public ResponseEntity getCoreInfo(@PathVariable("variantCoreString") String variantCoreString,
-                                      @RequestParam(name = "species") String species,
-                                      @RequestParam(name = "assembly") String assembly,
-                                      HttpServletResponse response) throws IllegalArgumentException {
+    public ResponseEntity getCoreInfo(
+            @ApiParam(value = "Internal species-relative variant ID. Chromosome, start, reference allele and" +
```

Comment:
Same as in `RegionWSServerV2`.
```suggestion
            @ApiParam(value = "Chromosome, start, reference allele and" +
```